src/Microsoft.Health.Dicom.Blob/Features/Export/AzureBlobExportSink.cs

```diff
@@ -63,8 +63,8 @@ public async Task<bool> CopyAsync(ReadResult value, CancellationToken cancellati
             return false;
         }
 
-        using Stream sourceStream = await _source.GetStreamingFileAsync(value.Identifier.Version, value.Identifier.Partition.Name, cancellationToken);
-        BlobClient destBlob = _dest.GetBlobClient(_output.GetFilePath(value.Identifier));
+        using Stream sourceStream = await _source.GetStreamingFileAsync(value.Instance.VersionedInstanceIdentifier.Version, value.Instance.VersionedInstanceIdentifier.Partition, value.Instance.InstanceProperties.fileProperties, cancellationToken);
```

Comment:
We might be able to save some space if we create a temporary variable for `value.Instance` like `i`